pkg/webmiddleware/log.go

```diff
@@ -47,6 +59,14 @@ func Log(logger log.Interface) MiddlewareFunc {
 			r = r.WithContext(log.NewContext(ctx, requestLogger))
 			metrics := httpsnoop.CaptureMetrics(next, w, r)
 
+			if metrics.Code < 400 {
+				for _, regex := range ignoreRegexes {
+					if regex.MatchString(r.URL.Path) {
+						return
+					}
+				}
```

Comment:
I don't know if the performance penalty of matching multiple regexes on every request is worth the added benefit of filtering out some logs. I think it's better to start with exact matches, and perhaps extend to prefixes in the future.